src/app.jsx

```diff
@@ -17,7 +17,7 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 import React, { useState, useEffect } from 'react';
-import { AlertGroup, Button } from '@patternfly/react-core';
+import { AlertGroup, Button, Progress, ProgressMeasureLocation } from '@patternfly/react-core';
```

Comment:
I think "show download progess" might better be squashed into the previous commit. If you want to keep it separate, please clarify the commit message, like "Show RHEL image download progress".